dll/win32/comctl32/lang/comctl_Hr.rc

```diff
@@ -0,0 +1,86 @@
+/*
+ * PROJECT:     ReactOS MessageBox dialog
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Croatian resource file
+ * TRANSLATORS: Copyright 2022 Andrej Bartulin <andrej.bartulinvr@gmail.com>
+ */
+
+LANGUAGE LANG_CROATIAN, SUBLANG_DEFAULT
```

Comment:
This thing has broke the build. :sweat_smile: Because Wine already has `comctl_Sr.rc` with `LANGUAGE LANG_SERBIAN, SUBLANG_DEFAULT` - the language identifier is exactly the same.

It probably should be `SUBLANG_CROATIAN_CROATIA`, but there are also different dialects: https://wiert.me/2015/02/11/readable-table-from-msdn-topic-language-identifier-constants-and-strings/#NotesPrim

Reply:
Added ` SUBLANG_CROATIAN_CROATIA` to comctl_Hr.rc.
Also, will I send this update to to Wine?

Reply:
It was also a problem with their Serbian translation, I fixed it in fe452a1d.

Reply:
Do I need to fix comdlg32 too?

Reply:
Ideally yes, as it turns out.

Reply:
I will but do I need new Pull Request for that